<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningProfilactic</class>
 <widget class="QDialog" name="PlanningProfilactic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Планирование профилактических мероприятий</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="2">
    <widget class="QSpinBox" name="sbYear">
     <property name="minimum">
      <number>2017</number>
     </property>
     <property name="maximum">
      <number>2099</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="CDbComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QLabel" name="lblNewPlanning">
     <property name="text">
      <string>Запланировать проведение мероприятий для</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddProf">
     <property name="text">
      <string>Профосмотров взрослого населения</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Начать</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QComboBox" name="cmbMonthTo">
     <property name="currentIndex">
      <number>11</number>
     </property>
     <item>
      <property name="text">
       <string>январь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>февраль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>март</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>апрель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>май</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>июнь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>июль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>август</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>сентябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>октябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ноябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>декабрь</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QComboBox" name="cmbMonthFrom">
     <item>
      <property name="text">
       <string>январь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>февраль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>март</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>апрель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>май</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>июнь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>июль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>август</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>сентябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>октябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ноябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>декабрь</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddDisp3">
     <property name="text">
      <string>Диспансеризации взрослого населения раз в 3 года</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QCheckBox" name="chkQuantityWorkAge">
     <property name="text">
      <string>Учитывать трудоспособное/нетрудоспособное население</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lblMonthFrom">
     <property name="text">
      <string>Месяц с</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="lblMonthTo">
     <property name="text">
      <string>Месяц по</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblYear">
     <property name="text">
      <string>Год проведения мероприятий</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddDisp1">
     <property name="text">
      <string>Диспансеризации взрослого населения ежегодной</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QCheckBox" name="chkCovidPriority">
     <property name="text">
      <string>Приоритет планирования переболевших COVID-19</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDbComboBox</class>
   <extends>QComboBox</extends>
   <header>library/DbComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
